Nozomi fix potential incidents loss when max page number reached (#39331)
demisto/content: Packs/NozomiNetworks/ReleaseNotes/1_0_19.md

#### Integrations

##### Nozomi Networks

> - Fixed potential loss of incidents whenit is reached the last page number

Reviewer: ```suggestion
- Fixed an issue that could cause incidents to be lost when reaching the last page.
```
